catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -21,6 +22,38 @@
     raise
 
 
+def _create_property(property_name, read_only=False, stopped_acquisition=True):
```

Comment:
I didn't point this out in the AV PR, but since this function `_create_property()` is specific to Hamamtsu cameras because of the getters and setters, lets call `property_name` -> `hamamatsu_property_name` instead. It's a bit long, but helps for understanding.

Also, could I maybe ask you to do a code-sneak and also replace that variable in the AV service with `av_property_name`?

Reply:
Done for both services ;)